code/__defines/items_clothing.dm

```diff
@@ -77,7 +77,8 @@
 #define BLOCK_GAS_SMOKE_EFFECT     0x10 // Blocks the effect that chemical clouds would have on a mob -- glasses, mask and helmets ONLY! (NOTE: flag shared with ONESIZEFITSALL)
 #define FLEXIBLEMATERIAL           0x20 // At the moment, masks with this flag will not prevent eating even if they are covering your face.
 #define COVER_PREVENT_MANIPULATION 0x40 // Only clothing with this flag will prevent manipulation under it. Its for space suits and such, unlike from usual Bay12 rules of clothing manipulation.
-
+#define DRAG_N_DROP_UNEQUIP        0x80 // Allow you put intems in hands with drag and drop
```

Comment:
use DRAG_AND_DROP_UNEQUIP instead
it's not good to use colloquialisms or slang in code